app/components/buttons_component.rb

```diff
@@ -8,4 +8,16 @@ def initialize(form:)
   end
 
   attr_reader :form
+
+  def submit_button_label
+    work_in_reviewed_coll? ? 'Submit for approval' : 'Deposit'
+  end
+
+  private
+
+  def work_in_reviewed_coll?
+    return false if form.object.instance_of?(CollectionForm)
+
+    Collection.find(form.object.collection_id).review_enabled?
```

Comment:
per Mike, there will always be a collection for a work.

Reply:
```suggestion
    form.object.collection.review_enabled?
```

Reply:
I tried this before, and I tried it again just now:

```
     ActionView::Template::Error:
        undefined method `collection' for #<WorkForm:0x00007fb5d028a040>
        Did you mean?  collection_id
      # ./app/components/buttons_component.rb:22:in `work_in_reviewed_coll?'
      # ./app/components/buttons_component.rb:13:in `submit_button_label'
      # ./app/components/buttons_component.html.erb:4:in `call'
```

Reply:
sorry, try `form.object.model.collection`

Reply:
yes - that worked!